src/main/java/org/broadinstitute/consent/http/util/InstitutionUtil.java

```diff
@@ -38,7 +38,12 @@ private ExclusionStrategy getSerializationExclusionStrategy(Boolean isAdmin) {
       @Override
       public boolean shouldSkipField(FieldAttributes field) {
         String fieldName = field.getName();
-        return !isAdmin && !(fieldName.equals("id") || fieldName.equals("name") || fieldName.equals("signingOfficials"));
+        
+        return !isAdmin && !(fieldName.equals("id")
+                || fieldName.equals("name")
+                || fieldName.equals("signingOfficials")
+                || fieldName.equals("displayName")
+                || fieldName.equals("userId"));
```

Comment:
It looks like we need to allow for `email` as well based on the mocks and [DUOS-1892](https://broadworkbench.atlassian.net/browse/DUOS-1892) (sorry about that!)

Reply:
good catch!